docs/guidelines/platforms/android/guideline_percievable_android.md

```diff
@@ -78,6 +111,18 @@ For all live media in the application, captions should be available through the
 
 The second approach is more customizable and can offer more flexibility than embedded captions, though it requires additional client-side work. For further details, please refer to the [Captions support for prerecorded media (WCAG 1.2.1 and 1.2.2 - Level A) section](#captions-support-for-prerecorded-media-wcag-121-and-122---level-a) and the [ExoPlayer documentation on live streaming](https://developer.android.com/media/media3/exoplayer/live-streaming), which provides more information about HTTP Live Streaming (HLS) on the Android platform.
 
+### Audio Description for Prerecorded Media (WCAG 1.2.5 - Level AA)
+
+This guideline covers [1.2.5 Audio Description (Prerecorded) - Level AA](https://www.w3.org/WAI/WCAG22/quickref/#audio-description-prerecorded) of the WCAG standard.
+
+As a part of this guideline, it is important to provide an audio description for prerecorded video content.
+
+:white_check_mark: **Success criteria**
+
+This guideline closely aligns with the [Audio Description or Media Alternative (WCAG 1.2.3 - Level A)](#audio-description-or-media-alternative-wcag-123---level-a), which states that video content must include an audio description. Unlike the previous guideline, which allows for alternatives, this requirement emphasizes the need for a dedicated audio track that describes the video content. This description can be integrated into the main audio track or provided as a separate audio track.
+
```

Comment:
Not sure if I understand correctly, but if you are using an audio alternative are you satisfying an A or AA level? If you comply with AA-level guidelines by adding audio to the video, I would suggest we just add it to this section.

Reply:
As far as I understand it. 1.2.3 says that you can use audio description track or any other noted alternatives. While 1.2.5 requires only audio description track. In that sense 1.2.5 is more stricter in regulations. But if you implemented for 1.2.3 the audio track, you are automatically satisfying 1.2.5, which means you are Level AA compliant.

Reply:
@Aksi0m yes, that is right.

Reply:
> But if you implemented for 1.2.3 the audio track, you are automatically satisfying 1.2.5, which means you are Level AA compliant.

IMO it would make sense then to move the audio description to 1.2.5 and keep other alternatives only for 1.2.3. Since, if you implement audio description you are anyway compliant with AA, not A level.

Reply:
Yes, that makes sense. But instead of removing it entirely, I’d add a reference in 1.2.3 pointing to the 1.2.5 guideline to avoid confusion, since _Audio Description_ is part of the 1.2.3 guideline title in WCAG.